app/file-detail/template.hbs

```diff
@@ -1,14 +1,21 @@
 {{title model.file.name}}
 {{quickfile-nav user=model.user onQuickfiles=false}}
 <div class='quickfiles-content row'>
-    <div class='col-sm-8'>
-        <h2>{{model.file.name}} <a id='versionLink' onclick={{action 'changeView'}}>(Version: {{mfrVersion}})</a></h2>
+    <div class='col-sm-5'>
+        <h2>{{model.file.name}} <a id='versionLink' onclick={{action 'changeViewPanel' 'revisionsPanel' 'revisionBtn'}}>(Version: {{mfrVersion}})</a></h2>
     </div>
-    <div class='col-sm-4'>
+    <div class='col-sm-7'>
         <div id='toggleBar' class='pull-right'>
             <div class='btn-toolbar m-t-md'>
+                {{#if edit}}
```

Comment:
it would be better to have a separate property for delete perms, e.g. `canDelete`

You could simply make it an alias of `canEdit`, e.g. `computed.alias('canEdit')`

This makes it so the view doesn't have to know that delete perms == edit perms